composer.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "networkrailbusinesssystems/activity-log",
-    "description": "Ahhhhh",
+    "description": "A library for documenting the activity log of a system",
```

Comment:
Not quite accurate. This library does not document anything. It contains the common activity log components, controllers, views, etc, that support our systems.